src/dispatch/database/revisions/core/versions/2024-04-22_83ec59cd3e4c.py

```diff
@@ -0,0 +1,37 @@
+"""empty message
+
+Revision ID: 83ec59cd3e4c
+Revises: ed0b0388fa3f
+Create Date: 2024-04-22 14:19:17.541843
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '83ec59cd3e4c'
+down_revision = 'ed0b0388fa3f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
I'm not really sure any of these are actually needed? As they look like they are just removing and re-added.

Reply:
Done. Removed this file.